app/assets/stylesheets/admin/logs/_log_modal.scss

```diff
@@ -0,0 +1,71 @@
+#log-modal {
```

Comment:
Since we're including everything with @import, I think almost all SCSS files should technically start with an _. I will rename just for consistency's sake.